README.md

```diff
@@ -422,10 +422,10 @@ gcloud components install docker-credential-gcr --quiet
 ### Applications
 
 <details>
-  <summary>Slack</summary>
+  <summary>Google Chat</summary>
```

Comment:
Thanks for doing this!

For now, it is better to install both slack and google chat. Many dev teams still use slack. 

I can see us removing Slack in the next month or two, though!

Reply:
Also, it looks like the version of Chat it installs is no longer maintained by Google or a hacked version. Opening it gave me this warning:
<img width="1450" alt="image" src="https://github.com/vendasta/setup-new-computer-script/assets/303329/0807246d-0b44-412f-ae67-f8cf2674ecc7">

Reply:
Oh well, maybe this change can be made in the future!

Reply:
(I'll take a look at that cask install)